SpatialGDK/Documentation/content/get-started/gdk-template.md

```diff
@@ -0,0 +1,152 @@
+If you haven't already, follow our [Multiserver Shooter tutorial]({{urlRoot}}/content/get-started/dependencies) to try out the SpatialOS for Unreal GDK development experience by adding a new feature to a project and testing it across two servers.
+
+If you are ready to start developing your own game with the GDK, follow the steps below. 
+
+### Terms used on this page
+
+* `<Game>` - The directory that contains your project’s .uproject file and Source folder.
+* `<ProjectRoot>` - The directory that contains your `<Game>` directory.
+* `<YourProject>` - The name of your project and .uproject file (for example, `\<Game>\YourProject.uproject`).
+
+### Create a new project using the starter template
+
+After [building the Unreal Engine fork]({{urlRoot}}/content/get-started/build-unreal-fork), in **File Explorer**, navigate to `UnrealEngine\Engine\Binaries\Win64`and double-click UE4Editor.exe to open the Unreal Editor. 
+
+In the [Project Browser](https://docs.unrealengine.com/en-us/Engine/Basics/Projects/Browser) window, click the New Project tab and select the C++ tab, then select the **SpatialOS GDK Starter** template. Choose a suitable directory and name for your project, then click **Create Project**. 
+
+>Note: When you create a project, the Unreal Engine automatically creates a directory named after the project name you entered. This page uses `YourProject` as an example project name.
+
+![The Unreal Engine Project Browser]({{assetRoot}}assets/set-up-template/template-project-browser.png)
+
+*Image: The Unreal Engine Project Browser, with the project file path and project name highlighted.*
+
+After clicking **Create Project**, the Unreal Engine generates the necessary project files before closing the editor and automatically opening Visual Studio. 
+
+### Clone the GDK
+
+Now you need to clone the SpatialOS GDK for Unreal into your project.
+
+1. In **File Explorer**, navigate to the `<Game>` directory and create a `Plugins` folder in this directory.
+2. In a Git Bash terminal window, navigate to `<Game>\Plugins` and clone the [GDK for Unreal](https://github.com/spatialos/UnrealGDK) repository by running either:
+    * (HTTPS) `git clone https://github.com/spatialos/UnrealGDK.git`
+    * (SSH) `git clone git@github.com:spatialos/UnrealGDK.git`
+
+>**Note** You need to ensure that the root directory of the GDK for Unreal repository is called `UnrealGDK` so the file path is: `<Game>\Plugins\UnrealGDK\...`
+
+### Build the dependencies 
+
+To use the starter template, you must build the GDK for Unreal module dependencies and add the GDK to your project. 
+
+1. Open **File Explorer**, navigate to the root directory of the GDK for Unreal repository, and double-click **`Setup.bat`**. You may be prompted to sign into your SpatialOS account if you have not already.
+1. In the same directory, double-click **`<YourProject>`.sln** to open it with Visual Studio.
+1. In the Solution Explorer window, right-click on **`<YourProject>`** and select **Build**.
```

Comment:
I believe @m-samiec mentioned that before we build the project, we need to regenerate the project files since we have run the 'Setup.bat' script and altered the project file structure. To do this we would need to right click on the <YourProject>.uproject and click 'Generate Visual Studio Project files'. I'm not sure if we need to close visual studio before doing this. @m-samiec can you please confirm?